frontend/src/ui/Blocks.tsx

```diff
@@ -57,14 +59,60 @@ type TextProps = {
     content: string;
 };
 
+// We don't just want to allow everything, so here we explicitly list what we
+// support. Notably missing:
+//
+// - img (we might want to allow them but need to figure out storage first)
+// - headings (if we allow them, we need to map `#` to h2 or h3 so that they
+//   don't interfere with other headings on the page.)
+const ALLOWED_MARKDOWN_TAGS = [
+    "p", "blockquote", "pre", "ul", "ol", "li", "a", "em", "strong", "code", "hr",
+];
+
+const CODE_BACKGROUND_COLOR = "#f9f9f6";
+
+// We override some components emitted by the Markdown parser to add CSS.
+const MARKDOWN_COMPONENTS: Components = {
+    p: ({ node: _, ...props }) => <p css={{ margin: "16px 0" }} {...props} />,
```

Comment:
Btw. I think I slightly prefer this way of ignoring props. I also wonder why we can't just pass `node` along as well, though. :thinking:

Reply:
And that also means I'm not sure about the added lint option, for the record. x)

Reply:
So, this only works for one parameter. `{ node: _, className: _, ...rest }` does not work because of using the same name twice. So that would mean `node: _node` and `className: _className`. Mhhh. Is that still better in your opinion?

And yeah dunno what would happen if we pass node along. Like... we probably don't want to pass random props to components right? What if those components suddenly start using them?

Reply:
Hmm, yeah, those are good points. :thinking: Aight, I give up. :shrug: